### ISSUE TITLE
FR: SRC_URI="${P}.tar.gz"

### ISSUE THREAD

**Issue by jonasstein** [mm1ke/gentoo-scripts#58]
https://bugs.gentoo.org/571594

similar to
https://github.com/mm1ke/gentoo-scripts/issues/41

**Comment by mm1ke** [comment]
good idea, i can start with ```qgrep -N 'SRC_URI="${P'``` but there might be more. Should be easy to implement.
